pkg/gofr/config/godotenv.go

```diff
@@ -5,13 +5,15 @@ import (
 	"fmt"
 	"io/fs"
 	"os"
+	"strings"
 
 	"github.com/joho/godotenv"
 )
 
 const (
 	defaultFileName         = "/.env"
 	defaultOverrideFileName = "/.local.env"
+	splitParts              = 2
```

Comment:
Are you still using this?

Maybe unused linter could be added to golangci-lint

Reply:
I have removed the const.

thankyou